cmd/statusd/main.go

```diff
@@ -290,16 +295,19 @@ Options:
 // stops the node. It times out after 5 seconds
 // if the node can not be stopped.
 func haltOnInterruptSignal(nodeManager common.NodeManager) <-chan struct{} {
+
+	logger := log.New("package", "status-go/cmd/statusd")
```

Comment:
hmm, shouldn't we make it a global variable for this file instead?

Reply:
Yeah, I was kinda thinking the same thing. It appears go-ethereum appears to log in this manner (initializing loggers within function scope), but I think package scope might make more sense if you guys are comfortable with that.